cmd/del/route53/cmd.go

```diff
@@ -0,0 +1,100 @@
+package route53
+
+import (
+	"fmt"
+	"github.com/aws/aws-sdk-go/service/route53"
+	"github.com/jharrington22/aws-resource/pkg/arguments"
+	"github.com/jharrington22/aws-resource/pkg/aws"
+	logging "github.com/jharrington22/aws-resource/pkg/logging"
+	rprtr "github.com/jharrington22/aws-resource/pkg/reporter"
+	"github.com/spf13/cobra"
+)
+
+var (
+	zoneId string
+)
+
+var Cmd = &cobra.Command{
+	Use:   "route53",
+	Short: "Delete route53 resources",
+	Long: `Delete route53 resources"
+aws-resource delete route53.`,
+	RunE: run,
+}
+
+func run(cmd *cobra.Command, args []string) (err error) {
+
+	reporter := rprtr.CreateReporterOrExit()
+	logging := logging.CreateLoggerOrExit(reporter)
+
+	awsClient, err := aws.NewClient().
+		Logger(logging).
+		Profile(arguments.Profile).
+		RoleArn(arguments.RoleArn).
+		Build()
+
+	if err != nil {
+		reporter.Errorf("Unable to build AWS client")
+		return err
+	}
+
+	if err != nil {
+		reporter.Errorf("Failed to describe regions")
+		return err
+	}
+
+	if zoneId != "" {
+		err := deleteHostedZoneId(awsClient, zoneId)
+		if err != nil {
+			reporter.Errorf("Unable to delete hosted zone: %s", err)
+		}
+		reporter.Infof("Hosted zone %s deleted", zoneId)
```

Comment:
Perhaps this line should move into `deleteHostedZoneId` so we get an indication of deleted zones when deleting all?